cobbler/actions/litesync.py

```diff
@@ -215,3 +215,27 @@ def remove_single_system(self, name: str):
             filename = system_record.get_config_filename(interface=name)
             utils.rmfile(os.path.join(bootloc, "pxelinux.cfg", filename))
             utils.rmfile(os.path.join(bootloc, "grub", filename.upper()))
+
+    def add_single_menu(self, rebuild_menu=True):
+        """
+        Sync adding a single menu.
+
+        :param name: The name of the profile.
+        :type name: str
+        :param rebuild_menu: Whether to rebuild the grub/... menu or not.
+        :type rebuild_menu: bool
+        """
+        if rebuild_menu:
+            self.sync.tftpgen.make_pxe_menu()
+
+    def remove_single_menu(self, rebuild_menu=True):
+        """
+        Sync removing a single menu.
+
+        :param name: The name of the profile.
+        :type name: str
+        :param rebuild_menu: Whether to rebuild the grub/... menu or not.
+        :type rebuild_menu: bool
+        """
+        if rebuild_menu:
+            self.sync.tftpgen.make_pxe_menu()
```

Comment:
It looks like the setting and getting the real menu is missing here?

Reply:
Updating the boot menu should never happen after adding a menu collection item, so `add_single_menu` has now been removed.

remove_single_menu is called from the `cobbler_collections.menus.remove()` that had errors. Fixed.